src/main/java/com/github/fge/jsonpatch/OmitOperation.java

```diff
@@ -0,0 +1,54 @@
+package com.github.fge.jsonpatch;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.MissingNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.fge.jackson.JsonNumEquals;
+import com.github.fge.jackson.jsonpointer.JsonPointer;
+import com.google.common.base.Equivalence;
+import com.google.common.collect.Iterables;
+
+/**
+ *
+ */
+public class OmitOperation extends PathValueOperation {
+
+    private static final Equivalence<JsonNode> EQUIVALENCE
+        = JsonNumEquals.getInstance();
+    @JsonCreator
+    public OmitOperation(@JsonProperty("path") final JsonPointer path,
+                         @JsonProperty("value") final JsonNode value)
+    {
+        super("omit", path, value);
```

Comment:
this makes me think we have the "OpKey" hanging off the PathValueOperation (and ideally all the way up to JsonPatchOperation) which means we should be able to use the "omit" here for the key in the hashMap instead of double specifying it.